src/pykx/pandas_api/pandas_meta.py

```diff
@@ -311,3 +311,18 @@ def agg(self, func, axis=0, *args, **kwargs): # noqa: C901
             return data
         else:
             return (q('{(flip enlist[`function]!enlist x)!y}', keyname, data))
+
+    @api_return
+    def isna(self):
+        tab = self
+        return q.null(tab)
+
+    @api_return
+    def isnull(self):
+        tab = self
+        return tab.isna()
```

Comment:
👍 The Pandas API clearly states that this is just an alias, so this is perfectly fine. Use `self.isna()` instead, as in the previous function.